cv.tex

```diff
@@ -150,102 +149,70 @@ \subsection{Programming}
+\location{Nov 2022 – Nov 2024 | Seattle, WA}
 \vspace{\topsep} % Hacky fix for awkward extra vertical space
 \begin{tightitemize}
-\item 52 out of 2500 applicants chosen to be a KPCB Fellow 2014.
+\item Lead Incident Responder for the Detections and Response Team (DART), serving two public security incidents \textbf{\href{https://dropbox.tech/security/changing-how-we-identify-malicious-urls-in-shared-documents}{[1 - Malicious Links]}} \textbf{\href{https://sign.dropbox.com/blog/a-recent-security-incident-involving-dropbox-sign}{[2 - Dropbox Sign]}}
+\item Transitioned security case management operations and metrics from Jira to Tines
+\item Co-developed and monitored insider risk detections for multiple Reduction in Force (RIF) events
+\item Lead emergent issues with Audit, Compliance, Risk teams, documenting playbooks and interaction outcomes
+\item Mentored several great security engineers into their first enterprise incident response role
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
 
-\runsubsection{Google}
-\descript{| Software Engineering Intern}
+\runsubsection{Amazon.com}
+\descript{Sr. SecEng (L5-6), Detections \& Monitoring Team}
 
-\location{May 2013 – Aug 2013 | Mountain View, CA}
+\location{July 2018 – June 2022 | Seattle, WA}
 \begin{tightitemize}
-\item Worked on the YouTube Captions team in primarily vanilla Javascript and Python to plan, design and develop the full stack implementation of a new framework to add and edit Automatic Speech Recognition captions.
-\item Created a backbone.js-like framework for the Captions editor.
-\item All code was reviewed, perfected, and pushed to production.
+\item Founded a team producing security alarms from GuardDuty, CloudTrail, and Linux endpoint telemetry sources
+\item Built, operated, and deprecated a multi-tenant cloud-native detection engine identifying security events from hundreds of thousands of AWS accounts
+\item Integrated our detections service into the Threat Intelligence Platform to communicate Mitre ATT\&CK coverage
+\item Standardized alarm documentation through the \textbf{\href{https://blog.palantir.com/alerting-and-detection-strategy-framework-52dc33722df2}{Alerting and Detection Strategy (ADS) Framework}}
+\item Detections workstream lead during the 2021 \textbf{\href{https://aws.amazon.com/blogs/opensource/hotpatch-for-apache-log4j/}{Log4Shell}} and \textbf{\href{https://blog.twitch.tv/en/2021/10/15/updates-on-the-twitch-security-incident/}{Twitch}} security incidents
+\item Accomplished interviewer with 100+ combined phone screen, in-person, or virtual interviews performed
+\item Promoted to SecEng 3 (L6) in July 2021
 \end{tightitemize}
 
 \sectionspace % Some whitespace after the section
 
 %------------------------------------------------
 
-\runsubsection{Phabricator}
-\descript{| Open Source Contributor \& Team Leader}
+\runsubsection{Amazon.com}
```

Comment:
(JC):
- Use an en dash instead of hyphen in your date range
- Past tense across the board. “Responded to security incidents from across Amazon’s retail business and subsidiaries…,” “Triaged, negotiated, and resolved responsible…,” “Co-led the on-site network…”
- “PoS” —> “point of sale”
- How important is listing when you were promoted?

Reply:
1. forever learning latex:
- Hyphen: `-`
- En-dash: `--`
- Em-dash: `---`
2. ACK
3. ACK
4. included to detail the role change over time, not important // definitely not a top line item